### PR TITLE
Add workflow to verify examples

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
PTAL @subfuzion

File: .github/workflows/verify_examples.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    shell: bash
+env:
+  PUB_ENVIRONMENT: bot.github
+
+jobs:
+  verify_examples:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        example_directory: ["examples/hello"]
```
Comment (kevmoo) [inline .github/workflows/verify_examples.yml:21]: @subfuzion – you can just add every example to this variable – assuming they all have the same shape – and they'll get tested!

Comment (subfuzion) [reply .github/workflows/verify_examples.yml:21]: What if they have a different shape, like `cloudevent` handlers? What needs to get updated for that?

Comment (subfuzion) [reply .github/workflows/verify_examples.yml:21]: Not sure I understand what you mean by that. Looks like the matrix has to be updated for each `example/<x>` directory and tests for the example have to pass. Is there something else?

**Review by subfuzion — COMMENTED** [review]
Just the one question since it wasn't clear to me what you meant with your comment, but don't let that hold this up.

**Review by subfuzion — APPROVED** [review]

File: .github/workflows/verify_examples.yml (hunk 1 above)
Comment (kevmoo) [reply .github/workflows/verify_examples.yml:21]: That's true. It has to be updated. But it's one entry in one place in one file to get all of the verification we want – without adding little 💩  in every directory

**Pull request merged**